docs/deploy/create_server.rst

```diff
@@ -39,26 +39,72 @@ Hetzner
 
 .. note::
 
-   Hetzner dedicated servers are physical servers, and are commissioned to order. Pay attention to any wait times displayed during the setup process, as some servers may not be available for several days.  
+   Hetzner dedicated servers are physical servers, and are commissioned to order. Pay attention to any wait times displayed during the setup process, as some servers may not be available for several days.
+
 
 #. Go to `Hetzner <https://www.hetzner.com/?country=us>`__
-#. Click "Dedicated", and navigate to choose a suitable server for your application. So far, we've used EX-Line servers. 
+#. Click "Dedicated", and navigate to choose a suitable server for your application. So far, we've used EX-Line servers.
 #. Click the "Order" button for the server that you've chosen
 
    #. Select a location; we've never had an issue with simply choosing the cheapest
    #. Select an operating system - "Ubuntu 18.04 LTS minimal"
-   #. Select any additional storage required 
+   #. Select any additional storage required
 
 #. Click "Order Now"
 #. (optionally) Select "Public key" in the "Server Login Details" section, and paste your SSH key in; this will be added to /root/.ssh/authorized_keys
 #. Click "Save"
-#. Review the contents of the cart, then click "Checkout" 
-#. Log in using OCP's credentials. This will happen automatically if you're already logged into Hetzner services. 
+#. Review the contents of the cart, then click "Checkout"
+#. Log in using OCP's credentials. This will happen automatically if you're already logged into Hetzner services.
 #. Check the "I have read your Terms and Conditions as well as your Privacy Policy and I agree to them." box
 #. Click "Order in Obligation"
 #. Wait until you receive an email notifying you that your server is ready, then proceed to deploying the service.
 
-   
+Some Hetzner servers only let you start on their recovery OS.
+
+If you were not able to select Ubuntu above, you will need to follow these additional steps:
+
+#. SSH into recovery image
+#. Test the server hardware
+
+  .. code-block:: bash
+
+    smartctl -t long /dev/<device>
+    smartctl -a /dev/<device>
```

Comment:
I was trying to make this section a prompt to test the hardware rather than an exact checklist.

Due to differences in hardware between servers, even with the same supplier, it's tricky to write a full testing checklist.

For example, with disks, the SMART values vary between makes, we are checking in Prometheus for specific Samsung values that won't exist on other disks. 

There are other bits of hardware to test as well, for instance hardware raid (if we start a server with it), but again this is hugely different depending on the RAID controller. 
If you were being very thorough you could run a full memory test - I wouldn't recommend this unless we suspected issues with the server memory as it takes hours/days to run.

To some extent we should trust Hetzner to not give us bad hardware equally now is the best time for issues to be identified. 

Does this make sense? 
How do you think it is best for the docs to reflect this?